features/scope-pseudo.yml

```
@@ -0,0 +1,7 @@
+name: :scope pseudo-class
```

I'm assuming we're only saying pseudo class because of `@scope`:

```suggestion
name: :scope (pseudo-class)
```